tests/txt.test.js

```diff
@@ -112,6 +112,44 @@ describe('validateCodeComments', () => {
   })
 })
 
+describe('The copyright date is not valid.', () => {
+  test('Copyright text date valid', async () => {
+    const doc = cloneDeep(baseTXTDoc)
+
+    doc.data.extractedElements.copyrightDates = [2025]
+
+    await expect(validateCopyrightDate(doc, { mode: MODES.NORMAL })).resolves.toHaveLength(0)
```

Comment:
Won't this test start breaking in 2026? The input should be modified to match the year in which the test is run instead.